Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nycplanning/library:ubuntu-latest
+FROM nycplanning/library:ubuntu-5f514f801976479945dfbdc3fc0b9ec864e2e810
```

Comment:
This is just a specific version of the docker image that you want to hold consistent?

Reply:
This i my fault for not looking at this sooner but its unclear to me what the impact of this is/will be and how this changes our workflow

Reply:
Yes this points to the working docker image that we've been using in our publishing workflows

Reply:
Hopefully as we talk through these pull requests it becomes more clear

Reply:
my understanding is this is a hack to not break anything that relies on the latest tag?

Reply:
no it is way to revert back to the last working thing. got it now.

Reply:
but I guess we also can't test if this tag will work until this merges in or if there are some other ways we can test this?